Realese Note for #15361: Implement volumetric fog support for both po… (#1745)
bevyengine/bevy-website: release-content/0.15/release-notes/15361_Implement_volumetric_fog_support_for_both_point_lights_and.md

> Volumetric fog was introduced in Bevy 0.14, where initially, only directional lights could interact with it. In Bevy 0.15, interactions with point lights and spot lights are also supported.

Reviewer: ```suggestion
Volumetric fog was [introduced in Bevy 0.14](https://bevyengine.org/news/bevy-0-14/#volumetric-fog-and-volumetric-lighting-light-shafts-god-rays). Initially, only directional lights could interact with it. In Bevy 0.15, point lights and spot lights work with it too.
```
